src/dateRange.ts

```diff
@@ -0,0 +1,19 @@
+import { jsDateToDateTime } from './dateMath';
+
+export const utcWeekRange = (on: Date = new Date()): [Date, Date] => {
```

Comment:
I would [label the items in this tuple](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-0.html#labeled-tuple-elements):

```suggestion
export const utcWeekRange = (on: Date = new Date()): [startOfWeek: Date, endOfWeek: Date] => {
```